CS1/Structural Recursion-Linked Lists/Spotify_Playlist_Solution_Version.py

```diff
@@ -10,6 +10,7 @@
 """
 from dataclasses import dataclass
 from typing import Any, Union
+from time import sleep
 
 
 # TODO: Create the dataclass for you linked node structure.
```

Comment:
Instead of using a dictionary to represent functions, it feels more natural to use a dataclass. The value of the LinkedNode should then be of "Song" time as opposed to "Any."